boards/atmel/sam/sam_v71_xult/Kconfig.defconfig

```diff
@@ -8,15 +8,11 @@ if ETH_SAM_GMAC
 
 # Read MAC address from AT24MAC402 EEPROM
 
-config ETH_SAM_GMAC_MAC_I2C_EEPROM
+config EEPROM
 	default y
-	select I2C
 
-config ETH_SAM_GMAC_MAC_I2C_INT_ADDRESS
-	default 0x9A
-
-config ETH_SAM_GMAC_MAC_I2C_INT_ADDRESS_SIZE
-	default 1
+config NVMEM
+	default y
```

Comment:
NVMEM, should be selected by the ethernet driver, if it has nvmem-cell is set, similar to the gpios being activated by the ethernet phys, that have a reset or irq gpio. (or at least implied)